### PR TITLE
Installations assignment0

### PR DESCRIPTION
**Pull request by jwoolbright23**

### CODE REVIEW

File: content/assignments/hello-world/test-and-commit/_index.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+title: "2. Test and Commit"
```
Comment (codinglikeagirl42) [inline content/assignments/hello-world/test-and-commit/_index.md:2]:
```suggestion
title: "3. Test and Commit"
```

File: content/assignments/hello-world/code-refactor/_index.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+---
+title: "1. Hello, World!"
```
Comment (codinglikeagirl42) [inline content/assignments/hello-world/code-refactor/_index.md:2]:
```suggestion
title: "2. Hello, World!"
```

File: content/installations/install-visual-studio/_index.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+Before we start coding on our computer, we need to make sure we have the right tools! Programmers use **integrated development environments** (IDE) to write and run their code. The development environment we will be using for this class is Visual Studio Code
+.
+In addition to simply writing and running code, Visual Studio Code has tools that recognize errors in our code and has an integrated terminal so we can navigate through our filesystem to find the files that need our attention.
+
+{{% notice blue Note "rocket" %}}
+Visual Studio Code is very customizable. Once you have everything set up, you can take additional steps to personalize your workspace such as changing the theme.
+{{% /notice %}}
+
+Go to the Visual Studio Code [download page](https://code.visualstudio.com/download/) and download the appropriate version for your operating system.
+
```
Comment (codinglikeagirl42) [inline content/installations/install-visual-studio/_index.md:19]: Suggestion add a note for Mac use to make sure to download the correct version depending on the chip your mac has.  M1 and M2 are silicon chips.

File: content/assignments/hello-world/project-intro/_index.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+If you haven't done so already, create a [GitHub account](https://github.com/join?ref_cta=Sign+up&ref_loc=header+logged+out&ref_page=%2F&source=header-home).
+
+### Assignment Invitation
+
+{{% notice blue Tip "rocket" %}}
+Before starting on any assignment, make sure you are signed into your GitHub account!
+{{% /notice %}}
+
+First, find `Assignment #0` in Canvas and click on the starter code repo link. Fork the repository to your personal profile, copy the repo’s URL for cloning, and clone the project to a directory on your machine.
+
```
Comment (codinglikeagirl42) [inline content/assignments/hello-world/project-intro/_index.md:48]: Suggestion add a section about enabling actions for the autograding.  Include brief description of how it works.

File: content/installations/install-visual-studio/_index.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+Now every time you open the terminal, it will default to bash!
+
+{{% notice blue Note "rocket" %}}
+1. If Git Bash is not an option when attempting to change the terminal profile, try downloading the [Github Pull Requests and Issues extension](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github).
+1. This extension comes with terminal integration.
+1. Restart Visual Studio Code and then go into the Command Palette again to change the terminal profile.
+{{% /notice %}}
+
+Please note that Markdown rendering can vary depending on the platform or Markdown editor you are using.
```
Comment (codinglikeagirl42) [inline content/installations/install-visual-studio/_index.md:37]: Suggestion add a note with resources to explore vsc -> https://code.visualstudio.com/docs

File: content/assignments/hello-world/test-and-commit/_index.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+1. `git status`
+
+![Image of terminal window after running the git status command](pictures/git-status.png?classes=border)
+
+1. `git add .`
+
+![Image of terminal window after running the git add . command](pictures/git-add.png?classes=border)
+
+1. `git commit -m "Hello world!"`
```
Comment (codinglikeagirl42) [inline content/assignments/hello-world/test-and-commit/_index.md:43]:
```suggestion
1. `git commit -m "Commit message"`
```

Comment (codinglikeagirl42) [reply content/assignments/hello-world/test-and-commit/_index.md:43]: Suggestion change the above and provide a brief description of what a commit message is.  For example: git commit -m"Updated the hello function"

File: content/installations/install-node/_index.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
+---
+title: "Installing Node"
+date: 2021-10-01T09:28:27-05:00
+draft: false
+weight: 5
+---
+
```
Comment (codinglikeagirl42) [inline content/installations/install-node/_index.md:7]: Suggestion - be very specific about which node version they should install - as upgrading to a new version may cause security issues.

**Review by codinglikeagirl42 — APPROVED** [review]
Few suggestions to address before merging.

File: content/installations/install-visual-studio/_index.md (hunk 3 above)
Comment (jwoolbright23) [reply content/installations/install-visual-studio/_index.md:19]: Added note within book for this.

File: content/assignments/hello-world/project-intro/_index.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+1. Sign up for a GitHub account.
+1. Change the code in Visual Studio Code.
+1. Commit the code.
+1. Push the code to your assignment repository.
+1. Make sure your assignment passes the test.
+
+### GitHub
+
+If you haven't done so already, create a [GitHub account](https://github.com/join?ref_cta=Sign+up&ref_loc=header+logged+out&ref_page=%2F&source=header-home).
+
```
[the remark this answers is not in the document]
Comment (jwoolbright23) [reply content/assignments/hello-world/project-intro/_index.md:40]: Added the above note to curriculum

File: content/assignments/hello-world/project-intro/_index.md (hunk 4 above)
Comment (jwoolbright23) [reply content/assignments/hello-world/project-intro/_index.md:48]: There is a description of github actions in the test-and-commit chapter. I believe this should be adequate for the learners to work through the assignment and learn a bit more about github actions. I will add a link to the actions documentation on github for additional support.

File: content/installations/install-visual-studio/_index.md (hunk 5 above)
Comment (jwoolbright23) [reply content/installations/install-visual-studio/_index.md:37]: Added to curriculum

**Pull request merged**